### ISSUE TITLE
Data Processing

### ISSUE THREAD

**Issue by haonan3** [wengong-jin/nips17-rexgen#4]
Hi,

Thanks for your work. And recently I am trying to reproduce the result of your paper.

I am a little bit confused about the dataset. There are 1,808,938 records in Lowe’s grants set, but I noticed you selected 480K records from them.

Could you please publish the data processing code or describe the algorithm?

Thanks in advance.

**Comment by wengong-jin** [comment]
Hi,

I think the dataset you mentioned (with 1.8M records) is 1976-2013_USPTOgrants_CML.7z from https://bitbucket.org/dan2097/patent-reaction-extraction/downloads/ 

Following previous work (Coley et al., 2017), we used its filtered version (1976-2013_USPTOgrants_reactionSmiles_feb2014filters.7z in that repo), which contains 1M records. We found that this dataset is still quite noisy. It contains lots of duplicates (after removing atom-map), and some atom mappings are incorrect. We removed these records from training.
